### PR TITLE
Organizations create

### PR DESCRIPTION
**Pull request by Gklmvch**
Older branch than [events_create]
Therefore some bugs can be fixed in [events_create] branch

### CODE REVIEW

File: app/controllers/organizations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  def new
+    @organization = Organization.new
+  end
+
+  def edit
+    @organization = Organization.find(params[:id])
+  end
+
+  def create
+    set_user
```
Comment (vmakoed) [inline app/controllers/organizations_controller.rb]: Move set_user to `before_action`

File: app/views/users/_organization.html.slim [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.col-sm-6.col-md-4.col-lg-3
+  .card.border-success.mb-3 style=("max-width: 18rem;")
```
Comment (vmakoed) [inline app/views/users/_organization.html.slim]: Do not use inline styles. Move them to a css (`organizations.css`) and create a class/id

File: app/controllers/organizations_controller.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
+class OrganizationsController < ApplicationController
+  def index
+    @organizations = Organization.order('name ASC')
+  end
+
+  def show
+    @organization = Organization.find(params[:id])
```
Comment (vmakoed) [inline app/controllers/organizations_controller.rb]: Extract this to a separate method
```ruby
before_action %i[show edit update destroy]

def find_user
  @organization = Organization.find(params[:id])
end
```

File: app/controllers/organizations_controller.rb [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+  def update
+    if @organization.update(organization_params)
+      redirect_to organization_path
+    else
+      redirect_to edit_organization_path
+    end
+  end
+
+  def destroy
```
Comment (vmakoed) [inline app/controllers/organizations_controller.rb]: Extra space after def
```suggestion
  def destroy
```

File: db/migrate/20181208170703_create_organizations.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class CreateOrganizations < ActiveRecord::Migration[5.2]
+  def change
+    create_table :organizations do |t|
+      t.string :name, null: false
+      t.text :summary
+      t.references :user, foreign_key: true
```
Comment (vmakoed) [inline db/migrate/20181208170703_create_organizations.rb]: timestamps?

File: app/views/organizations/new.html.slim [hunk 6]
```diff
@@ -0,0 +1,11 @@
+table.mx-auto style=("height: 90vh;")
```
Comment (vmakoed) [inline app/views/organizations/new.html.slim]: avoid using inline styles

File: app/views/organizations/new.html.slim [hunk 7]
```diff
@@ -0,0 +1,11 @@
+table.mx-auto style=("height: 90vh;") 
+  tbody
+    tr
+      td.align-middle
+        .jumbotron style=("width: 70vh;")
```
Comment (vmakoed) [inline app/views/organizations/new.html.slim]: avoid using inline styles

File: app/views/users/_user.html.erb [hunk 8]
```diff
@@ -0,0 +1,8 @@
+<tr>
+  <th scope="row"><%= user.first_name %></th>
```
Comment (vmakoed) [inline app/views/users/_user.html.erb]: move `<%= user.first_name %>` to a line below

Comment (Gklmvch) [reply app/views/users/_user.html.erb]: > move `<%= user.first_name %>` to a line below

Fix this and previous mistakes in [elements create] PR

**Pull request merged**